src/helpers/parseTranscriptionData.js

```diff
@@ -1,8 +1,13 @@
 function constructCoordinates(line) {
   const points = [];
+
   if (line && line.clusters_x && line.clusters_y) {
-    line.clusters_x.forEach((point, i) => {
-      points.push({ x: line.clusters_x[i], y: line.clusters_y[i] });
+    const lastPoint = line.clusters_x.length - 1
```

Comment:
I assume that since you're trusting the results from Tove, there _shouldn't_ be any conditions where...

1. line.clusters_x and line.clusters_y have different lengths
1. line.clusters_x will have at least (exactly?) 2 items

Reply:
Correct. There should always be the same amount of X and Y points from TOVE, since each annotation point will have an X and Y coordinate. Every project using TOVE/ALICE should only have two points, but using `length - 1` will still allow lines to render with the first and last points.